### PR TITLE
Use timestamp.sectigo.com for signing.

### PR DESCRIPTION
**Pull request by allexzander**
Signed-off-by: alex-z <blackslayer4@gmail.com>

### CODE REVIEW

File: defaults.inc.bat [hunk 1]
```diff
@@ -128,7 +128,7 @@ if "%P12_KEY%" == ""                        set P12_KEY=%PROJECT_PATH%\key\%APPL
 if "%P12_KEY_PASSWORD%" == ""               set P12_KEY_PASSWORD=
 
 if "%SIGN_FILE_DIGEST_ALG%" == ""           set SIGN_FILE_DIGEST_ALG=sha256
-if "%SIGN_TIMESTAMP_URL%" == ""             set SIGN_TIMESTAMP_URL=http://tsa.swisssign.net
+if "%SIGN_TIMESTAMP_URL%" == ""             set SIGN_TIMESTAMP_URL=http://timestamp.sectigo.com
```
Comment (mgallien) [inline defaults.inc.bat:131]: please use `https`

Comment (allexzander) [reply defaults.inc.bat:131]: @mgallien Sorry, but, this doesn't work **SignTool Error: Invalid Timestamp URL: https://timestamp.sectigo.com**

Comment (allexzander) [reply defaults.inc.bat:131]: @mgallien Also, when using old URL but with `https` as you recommend, same error shows up **SignTool Error: Invalid Timestamp URL: https://tsa.swisssign.net**

Comment (mgallien) [reply defaults.inc.bat:131]: OK

**Review by mgallien — APPROVED** [review]

**Pull request merged**